contributor-book/plugins.md

```diff
@@ -1081,15 +1081,18 @@ def send_error(id, error_msg, span):
 
 def handle_call(id, call_info):
     try:
-        input = call_info["input"]["Value"]["String"]
+        input = call_info["input"]["Value"][0]["String"]
```

Comment:
This is the only part that gives me pause. Seems odd that we'd have to get the 0th element, but maybe that's right?